src/sqlancer/timescaledb/TimeScaleDBProvider.java

```diff
@@ -0,0 +1,31 @@
+package sqlancer.timescaledb;
+
+import com.google.auto.service.AutoService;
+
+import sqlancer.DatabaseProvider;
+import sqlancer.common.query.ExpectedErrors;
+import sqlancer.postgres.PostgresGlobalState;
+import sqlancer.postgres.PostgresOptions;
+import sqlancer.postgres.PostgresProvider;
+import sqlancer.timescaledb.gen.TimeScaleDBCommon;
+
+@AutoService(DatabaseProvider.class)
+public class TimeScaleDBProvider extends PostgresProvider {
+    @SuppressWarnings("unchecked")
+    public TimeScaleDBProvider() {
+        super((Class<PostgresGlobalState>) (Object) TimeScaleDBGlobalState.class,
+                (Class<PostgresOptions>) (Object) TimeScaleDBOptions.class);
+    }
+
+    @Override
+    public String getDBMSName() {
+        getTimeScaleDBErrors(); // todo: invoke this to pass tests, remove this in the future
```

Comment:
I don't understand. Why do we need this?

Reply:
Some tests will not pass if we do not invoke `getTimeScaleDBErrors();` 

IMHO, it's not reasonable when a method that is not used locally results in an error. But since that is the case, I added this to avoid such error.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project sqlancer: Compilation failure: Compilation failure: 
[ERROR] [ecj] The compiler reported an error but has not written it to its logging
[ERROR] [ecj] The following line(s) might indicate the issue:
[ERROR]                                       ^^^^^^^^^^^^^^^^^^^^^^
[ERROR] The method getTimeScaleDBErrors() from the type TimeScaleDBProvider is never used locally
[ERROR] ----------
[ERROR] 1 problem (1 warning)
[ERROR] error: warnings found and -failOnWarning specified
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

Reply:
> IMHO, it's not reasonable when a method that is not used locally results in an error

Since the method is `private`, the compiler can infer that this is dead code, which is unexpected. I think we could just delete the method, since the expected errors should be specific to an action (e.g., inserting data).

Reply:
Got it. I make it public now